hack/print-workspace-status.sh

```diff
@@ -17,58 +17,10 @@ set -o errexit
 set -o nounset
```

Comment:
Can't we delete this file?

Reply:
Actually, yes, I think so.

Reply:
@ncdc I left the `print-workspace-status.sh` file as I wasn't sure what it was used for. But happy to delete it if it's not needed any more.

Reply:
Delete away!

Reply:
deleted!